app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,21 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        await this.loader.post('orders/resend-email', payload);
+        this.notify.success(this.l10n.t('Email confirmation has been sent to attendees successfully'));
+      } catch (error) {
+        this.notify.error(this.l10n.t('An unexpected error occurred.'));
```

Comment:
What was the error ? Could we display something better ? Like for example, if it was a ratelimitting error, show that it was due to it. Else show whatever the actual error sent by the backend is.

Reply:
@niranjan94 I have changed error handling, is it any better?